docs/playbooks/bas.md

```diff
@@ -1,20 +1,20 @@
 # BAS Playbooks
 
 ## Install BAS
```

Comment:
Could you add a description of what the playbook actually does, maybe a summary of the README.md: `Installs **IBM Behavior Analytics Services** on IBM Cloud Openshift Clusters (ROKS) and generates configuration that can be directly applied to IBM Maximo Application Suite.`